public/locales/en.json

```diff
@@ -133,7 +133,7 @@
         }
     },
     "editor": {
-        "placeholder": "← Start by entering a title here\n===\nVisit /features if you don't know what to do.\nHappy hacking :)",
+        "placeholder": "← Start by entering a title here\n===\nVisit /n/features if you don't know what to do.\nHappy hacking :)",
```

Comment:
Maybe talk about "the features page" instead, the `/n/` reads a bit odd in this sentence :)